Features/mu_config_ext_dep.yaml

```diff
@@ -1,4 +1,3 @@
-

```

Comment:
This is now a submodule under Features/CONFIG. You'll need to update the commit there.